src/app/containers/CpsFeaturesAnalysis/index.jsx

```diff
@@ -29,6 +52,20 @@ const StoryPromoLiFeatures = styled(StoryPromoLi)`
   &:last-child {
     padding: 0.5rem 0 0 0;
   }
+
+  ${() =>
```

Comment:
Same here

Reply:
Good spot, didn't realise you could write dynamic styles without a function 👍🏼